.github/workflows/swift.yml

```
@@ -0,0 +1,91 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Swift
+
+on:
+  push:
+    paths:
+      - '.github/workflows/swift.yml'
+      - 'ci/docker/**'
+      - 'ci/scripts/swift_*'
+      - 'swift/**'
+  pull_request:
+    paths:
+      - '.github/workflows/swift.yml'
+      - 'ci/docker/**'
+      - 'ci/scripts/swift_*'
+      - 'swift/**'
+
+concurrency:
+  group: ${{ github.repository }}-${{ github.head_ref || github.sha }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+env:
+  DOCKER_VOLUME_PREFIX: ".docker/"
+
+jobs:
+
+docker:
+    name: ${{ matrix.title }}
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        name:
+          - swift-5-7-1
+        include:
+          - name: swift-5.7.1
+            cache: conda-swift-5.7.1
+            image: ubuntu-swift
+            title: Swift ubuntu
+    env:
+      UBUNTU: ${{ matrix.ubuntu || 20.04 }}
```

It seems that `matrix.ubuntu` isn't defined.

```suggestion
    env:
      UBUNTU: ${{ matrix.ubuntu || 20.04 }}
```

---

Will remove matrix.ubuntu